src/applications/caregivers/definitions/caregiverUI.js

```diff
@@ -40,41 +37,51 @@ export default {
       'ui:title': 'Email Address',
       'ui:widget': 'email',
```

Comment:
The design has the email field as required.

Reply:
And there is an email UI definition you can build this off of: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/definitions/email.js